tools/pepsi/src/parsers.rs

```diff
@@ -175,6 +175,12 @@ impl FromStr for NaoNumberPlayerAssignment {
     }
 }
 
+impl Display for NaoNumberPlayerAssignment {
+    fn fmt(&self, formatter: &mut Formatter<'_>) -> fmt::Result {
+        self.nao_number.fmt(formatter)
+    }
+}
```

Comment:
You are providing an implementation detail as public functionality. It would be better that you just have this at the location where you are using it but not at this location. If I don't know your implementation, why would a number assignment only print the number?!

Reply:
Alright, I now changed it to output "nao_number:player_number".